src/ethereum/frontier/vm/instructions/__init__.py

```diff
@@ -43,9 +45,21 @@ class Ops(enum.Enum):
     # Computation Ops
     STOP = 0x00
 
+    # Sha3 Ops
+    SHA3 = 0x20
```

Comment:
```suggestion
    KECCAK = 0x20
```
maybe it's time?

Reply:
[We had exactly the same thought](https://github.com/ethereum/eth1.0-specs/pull/262#discussion_r670107898).

Reply:
woo woo woo!!!!

Reply:
https://media1.tenor.com/images/0f94a8ce6bcc9f02a0b12849f6fbe6be/tenor.gif?itemid=12543209